packages/@aws-cdk/aws-redshift-alpha/test/database-query-provider/privileges.test.ts

```diff
@@ -147,9 +149,27 @@ describe('update', () => {
     }));
   });
 
-  test('does not replace when privileges change', async () => {
```

Comment:
maybe im reading the diff wrong but seems like the old test got replaced. if thats necessary, can you let me know why? if not, i think we should keep the old test and add in your new ones as separate tests.

Reply:
The old test was removed, as it was more generic. The new tests are more fined tuned, specific to table names, ids and actions. The old test was used whenever the privilege changed. The test itself was probably generic since the function itself was very basic. However, the new functionality is more complex, so more fined tuned tests are required I think.